dist.java/java/logo-banner-override.css

```diff
@@ -0,0 +1,8 @@
+.navbar-brand img {
```

Comment:
Wonder if we shouldn't inline these styles in index.html to prevent ever accidentally deleting this file from a hack script or grunt task.